pom.xml

```diff
@@ -327,6 +328,32 @@
             </plugin>
 
             <plugin>
+                <groupId>com.mycila</groupId>
+                <artifactId>license-maven-plugin</artifactId>
+                <version>2.11</version>
+                <configuration>
+                    <header>src/main/license/header.txt</header>
+                    <excludes>
+                        <exclude>circle.yml</exclude>
+                        <exclude>jpmInstall.sh</exclude>
+                        <exclude>**/*.md</exclude>
+                        <exclude>bootstrap.min.css</exclude>
+                        <exclude>src/test/**</exclude>
+                        <exclude>**/LogicalOperatorRules.g4</exclude>
```

Comment:
the license maven plugin does not know this file extension, but I added a [custom mapping](supported-comment-types)